_locales/en/messages.json

```diff
@@ -629,6 +685,74 @@
       }
     }
   },
+  "styleInstallOverwrite" : {
+    "message": "'$stylename$' is already installed. Overwrite?\nVersion: $oldVersion$ -> $newVersion$",
+    "description": "Confirmation when re-installing a style",
+    "placeholders": {
+      "stylename": {
+        "content": "$1"
+      },
+      "oldVersion": {
+        "content": "$2"
+      },
+      "newVersion": {
+        "content": "$3"
+      }
+    }
+  },
+  "styleInstallNoName": {
+    "message": "Install this style into stylus?",
+    "description": "Confirmation when installing a style"
+  },
+  "styleInstallFailed": {
+    "message": "Failed to install userstyle!\n$ERROR$",
+    "description": "Warning when installation failed",
+    "placeholders": {
+      "error": {
```

Comment:
`$ERROR` should match "error"... Or, actually I'm not sure if placeholders are case-insensitive. @tophf?